build.gradle

```diff
@@ -86,13 +85,13 @@ uploadArchives {
 
                 developers {
                     developer {
-                        id = 'Ridhwana'
-                        name = 'Ridhwana Khan'
-                        email = 'ridhwana@smileidentity.com'
+                        id = 'Arlon'
+                        name = 'Arlon Mukeba'
```

Comment:
Same as the other PR let;s use support details here
id support
name Smile Identity
email support@smileidentity.com
etc

Reply:
👍🏽